README.md

```diff
@@ -7,61 +7,106 @@
 </a> <a href="https://goreportcard.com/report/sigs.k8s.io/kind"><img alt="Go Report Card" src="https://goreportcard.com/badge/sigs.k8s.io/kind" /></a>
 
 
-`kind` is a tool for running local Kubernetes clusters using Docker container "nodes".  
-`kind` is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests].
+`kind` is a tool for running local Kubernetes clusters using Docker container as "nodes".  
+`kind` is primarily designed for testing Kubernetes 1.11+.
+It initially targets the [conformance tests].
 
-If you have [go] and [docker] installed `go get sigs.k8s.io/kind && kind create cluster` is all you need!
+If you have [Go] and [Docker] installed, the following commands are all you need!
 
-<img src="https://gist.githubusercontent.com/BenTheElder/621bc321fc6d9506fd936feb36d32dd0/raw/7fe14e9d0929cab428929ca6c501abc990c07359/kind-create-cluster.gif" alt="2x speed `kind create cluster` demo" />
+```console
+go get -u sigs.k8s.io/kind
+kind create cluster
+export KUBECONFIG=$(kind get kubeconfig-path)
+```
+
+<img src="https://gist.githubusercontent.com/BenTheElder/621bc321fc6d9506fd936feb36d32dd0/raw/7fe14e9d0929cab428929ca6c501abc990c07359/kind-create-cluster.gif" alt="kind create cluster" />
 
 `kind` consists of:
  - Go [packages][packages] implementing [cluster creation][cluster package], [image build][build package], etc.
  - A command line interface ([`kind`][kind cli]) built on these packages.
  - Docker [image(s)][images] written to run systemd, Kubernetes, etc.
- - [`kubetest`][kubetest] integration also built on these packages (WIP)
+ - [`kubetest`][kubetest] integration also built on these packages (currently WIP).
+
+`kind` bootstraps each "node" with [kubeadm][kubeadm].
+For more details see [the design documentation][design doc].  
+
+**NOTE**: `kind` is still a work in progress.
+See [docs/todo.md] for a list of pending tasks.
+
+## Installation
+
+You can install `kind` by running:
+
+```console
+go get -u sigs.k8s.io/kind
+```
+
+## Usage
+
+To use `kind`, you need to [install Docker].
+Once you have Docker running you can create a cluster by running:
+
+```console
+kind create cluster
+```
+
+To point at the newly created cluster, you can run:
```

Comment:
I think we should rephrase this, something like "To configure your shell to use the newly created cluster ...", because "point at" doesn't quite seem ideal, and it is scoped to your shell, which some users might not catch.